src/fluree/db/api/transact.cljc

```diff
@@ -28,11 +29,14 @@
 (defn stage
   [db txn opts]
   (go-try
-   (let [txn-context (or (ctx-util/txn-context txn)
-                         (:context opts))
-         parsed-opts (parse-opts txn opts txn-context)
-         parsed-txn  (q-parse/parse-txn txn txn-context)]
-     (<? (connection/stage-triples db parsed-txn parsed-opts)))))
+    (let [txn*        (if (= :sparql (:format opts))
```

Comment:
i think it makes sense to convert this check into a function that takes the transaction and the format since it's used multiple times.